sebs/azure/azure.py

```
@@ -513,8 +575,40 @@ def enforce_cold_start(self, functions: List[Function], code_package: Benchmark)
     """
 
     def create_trigger(self, function: Function, trigger_type: Trigger.TriggerType) -> Trigger:
-        raise NotImplementedError()
+        from sebs.azure.triggers import QueueTrigger, StorageTrigger
 
+        azure_function = cast(AzureFunction, function)
+        resource_group = self.config.resources.resource_group(self.cli_instance)
+        storage_account = azure_function.function_storage.account_name
+
+        ret = self.cli_instance.execute(
+            ('az storage account show --resource-group {} --name {} --query id')
+            .format(resource_group, storage_account)
+        )
+        self.cli_instance.execute(
+            ('az role assignment create --assignee "{}" \
+              --role "Storage {} Data Contributor" \
+              --scope {}')
+            .format(
+                os.environ["AZURE_USER_PRINCIPAL_NAME"],
+                "Queue" if trigger_type == Trigger.TriggerType.QUEUE else "Blob",
+                ret.decode("utf-8")
```

plz check for the return value if it makes sense/not empty - fail early